Godeps/_workspace/src/k8s.io/kubernetes/cmd/kubelet/app/server.go

```diff
@@ -148,6 +148,9 @@ type KubeletServer struct {
 
 	KubeApiQps   float32
 	KubeApiBurst int
+
+	// Pull images one at a time.
+	SerializeImagePulls bool
```

Comment:
Isn't this going to be false by default now?  I don't see where the default value is set to true.